openpype/hosts/hiero/api/plugin.py

```diff
@@ -721,16 +722,28 @@ def convert(self):
         if self.rename:
             # rename track item
             self.track_item.setName(new_name)
-            self.tag_data["asset"] = new_name
+            self.tag_data["asset_name"] = new_name
         else:
-            self.tag_data["asset"] = self.ti_name
+            self.tag_data["asset_name"] = self.ti_name
             self.tag_data["hierarchyData"]["shot"] = self.ti_name
 
+        # AYON unique identifier
+        folder_path = "/{}/{}".format(
+            tag_hierarchy_data["hierarchy"],
```

Comment:
Is possible that `tag_hierarchy_data["hierarchy"]` is empty?

Reply:
Impossible.

Reply:
So it cannot happen that this ends up as `//sh0010`?